.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/inuitcss/inuitcss
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+      LC_ALL: en_US.UTF-8
+      LANG: en_US.UTF-8
+    docker:
+      - image: circleci/build-image:ubuntu-14.04-XXL-upstart-1189-5614f37
```

Comment:
Hm, do we really need such a big image? I’d be in favor of using a smaller distribution like arch or alpine…